frontend/src/main-components/StudyModeContent.js

```diff
@@ -1,46 +1,81 @@
 import React from "react";
-import RoundLights from "../sub-components/RoundLights";
-import FlashcardBack from "../flashcards/FlashcardBack";
-import Options from "../sub-components/CardOptions";
-import css from "./StudyModeContent.css";
+import styled from "@emotion/styled";
+import StudyModeQuiz from "../sub-components/StudyModeQuiz";
+import { nextQuizWord, updateWordQueues, getRound } from "./StudyModeGameHandler";
+
+const EndGame = styled.h1`
+  color: #d4af37;
+  font-size: 10rem;
+`;
 
 class StudyModeContent extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      isDataFetched: false,
       currentRound: 1,
-      totalRounds: 4,
+      quizWord: "none",
+      options: [],
+      correctAnswer: "none",
+      end: false,
     };
     this.optionSelected = this.optionSelected.bind(this);
   }
 
+  componentDidMount() {
+    // Grabs the first word in the game
+    const word = this.props.quiz.nextQuizWord();
+    const round = this.props.quiz.getCurrentRound();
+    this.setState({
+      quizWord: word.quizWord,
+      options: word.possibleResponses,
+      correctAnswer: word.correctAnswer,
+      currentRound: round
+     });
+  }
+
   optionSelected(event) {
     const selectedValue = event.currentTarget.value;
-    // TODO(esaracay): Check if the word was correct
-  }
+    let correct = "false";
```

Comment:
Don't use raw strings here; put the string into a constant and use the constant instead. This avoids typos/errors when a string is changed in one place but not the other.

Reply:
done.